p4-16/psa/PSA.mdk

```diff
@@ -739,8 +739,8 @@ packet contents and metadata when a packet begins egress processing.
 |                   | of ingress packet  | configuration      | of clone session       ||
 |                   |                    | of multicast group |                        ||
 +--------------+-------------+---------------+--------------+-----+
-| `packet_path`   | NORMAL_ | NORMAL_   | CLONE_I2E | CLONE_E2E |
-|                   | UNICAST | MULTICAST |           |           |
+| `packet_path`   | NORMAL | NORMAL   | CLONE_I2E | CLONE_E2E |
+|                 | UNICAST | MULTICAST |           |           |
```

Comment:
Might be nice to restore the underscore characters that were removed from after the word 'NORMAL_' here.

Reply:
Done